src/Enterspeed.Cli/Services/IngestService/IngestService.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System.Text;
+using System.Text.Json;
+using Microsoft.Extensions.Logging;
+
+namespace Enterspeed.Cli.Services.IngestService;
+
+public class IngestService : IIngestService
+{
+    private readonly HttpClient _httpClient;
+
+    private readonly ILogger<IngestService> _logger;
+    public IngestService(ILogger<IngestService> logger, ISettingsService settingsService)
+    {
+        _logger = logger;
+
+        _httpClient = new HttpClient
+        {
+            BaseAddress = new Uri(settingsService.GetSettings().IngestApiUri),
+        };
+    }
+
+    public async Task<bool> Ingest(string filePath, string apiKey, bool useFilenameAsId)
```

Comment:
Does the CLI support both raw and simplified formats?

Reply:
Simple for now, but I guess we could expand at some point in the future.